### PR TITLE
Text Extraction taskrunner

### PR DESCRIPTION
**Pull request by kbirk**
# Description

* Include a summary of the changes and the related issue. 
* Include relevant motivation and context.

Resolves #(issue)

### CODE REVIEW

File: packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/ExtractionService.java [hunk 1]
```diff
@@ -831,21 +832,52 @@ public Future<EquationExtraction> extractEquationsFromPDF(
...
+
+			final TextExtraction extraction = new TextExtraction();
+
+			extraction.documentText = output.getResponse().asText();
+
+			return extraction;
+		});
+	}
+
+	public Future<TextExtraction> extractTextFromPDFCosmos(
```
Comment (YohannParis) [inline packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/ExtractionService.java:874]: Why do we need this? As a backup?

File: packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/ExtractionService.java [hunk 2]
```diff
@@ -208,14 +209,14 @@ public ExtractPDFResponse runExtractPDF(
...
-			extractionResponse.assets = cosmosTextExtraction.assets;
-			extractionResponse.files = cosmosTextExtraction.files;
+			// wait for text extraction
+			final TextExtraction textExtraction = textExtractionFuture.get();
+			notificationInterface.sendMessage("Text extraction complete!");
+			log.info("Text extraction complete for document: {}", documentName);
+			extractionResponse.documentAbstract = textExtraction.documentAbstract;
+			extractionResponse.documentText = textExtraction.documentText;
+			extractionResponse.assets = textExtraction.assets;
+			extractionResponse.files = textExtraction.files;
```
Comment (YohannParis) [inline packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/ExtractionService.java:219]: Why adding all of those if only `documentText` is returned for now?

File: packages/text_extraction/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+###### DEV ONLY ######
+#VVVVVVVVVVVVVVVVVVVVV
+
+# These deps are installed only for use during local development
+
+# Install OpenJDK
+RUN apt-get update && apt-get install -y --no-install-recommends \
+	ocrmypdf \
+	openjdk-17-jdk-headless && \
+	rm -rf /var/lib/apt/lists/*
```
Comment (YohannParis) [inline packages/text_extraction/Dockerfile:13]: Is this up to date with the latest drama with debian and openJDK?

**Review by YohannParis — APPROVED** [review]

File: packages/text_extraction/Dockerfile (hunk 3 above)
Comment (dgauldie) [reply packages/text_extraction/Dockerfile:13]: looks like it

**Review by dvince2 — APPROVED** [review]

**Pull request merged**